src/sql/src/names.rs

```diff
@@ -1594,10 +1616,19 @@ impl<'a> Fold<Raw, Aug> for NameResolver<'a> {
         &mut self,
         stmt: CreateContinualTaskStatement<Raw>,
     ) -> CreateContinualTaskStatement<Aug> {
-        // TODO(ct): Insert a fake CTE so that using the name of the continual
-        // task in the inserts and deletes resolves.
-        let cte_name = normalize::ident(stmt.name.0.last().expect("TODO(ct)").clone());
-        self.ctes.insert(cte_name, LocalId::new(0));
+        // Insert a LocalId so that using the name of the continual task in the
+        // inserts and deletes resolves.
+        match normalize::unresolved_item_name(stmt.name.name().clone()) {
+            Ok(local_name) => {
+                assert!(self.continual_task.is_none());
+                self.continual_task = Some((local_name, LocalId::new(0)));
```

Comment:
At this point I don't think we need to hard code `LocalId(0)` here anymore and could do something a little more robust, like have some `next_local_id` field? Totally not necessary in this PR but maybe add a `// TODO`?

Reply:
Went ahead and did it, but it broke something about CTEs so ripped it out. Left the TODO